core/src/impl/Kokkos_QuadPrecisionMath.hpp

```diff
@@ -208,8 +208,7 @@ inline bool signbit(__float128 x) { return ::signbitq(x); }
 //</editor-fold>
 
 //<editor-fold desc="Mathematical constants __float128 specializations">
-namespace Kokkos {
-namespace Experimental {
+namespace Kokkos::numbers {
```

Comment:
I believe every Kokkos namespace starts with a capital letter (`Kokkos`, `Impl`, `Experimental`, `Test`, etc). I think it'll be better if we keep that consistent here as well.

Reply:
This was discussed at the developer meeting.  Consensus was for `Kokkos::numbers` lowercase for consistency with C++20.